windows.fnl

```diff
@@ -1,10 +1,6 @@
-(local {:filter filter} (require :lib.functional))
-;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;;
```

Comment:
Config is moved to the init function of this module which uses the table returned from config.fnl or private/config.fnl